.env

```diff
@@ -10,8 +10,17 @@ FLOW_INIT_ACCOUNTS=0
 CONTRACT_FUNGIBLE_TOKEN=0xee82856bf20e2aa6
 CONTRACT_FLOW_TOKEN=0x0ae53cb6e3f42a79
 CONTRACT_FUSD=0xf8d6e0586b0a20c7
+CONTRACT_FCL_CRYPTO=0xf8d6e0586b0a20c7
 FLOW_AVATAR_URL=https://avatars.onflow.org/avatar/
 
+# Default values. These should match locally running emulator.
+# Will only be used when wallet is started from source (npm run dev)
+FLOW_ACCOUNT_ADDRESS=0xf8d6e0586b0a20c7
+FLOW_ACCOUNT_KEY_ID=0
```

Comment:
`FLOW_ACCOUNT_KEY_ID` is already defined on line 8. Maybe just delete line 8?

Reply:
Yeah, also think .env should not be checked in. we should document requirements in .env.example and suggest copying to .env.local in README (may already be there)

Reply:
Oh yeah, I agree that `.env` shouldn't be committed.